src/screens/SendFunds/04-Summary.js

```diff
@@ -78,6 +79,10 @@ function SendSummary({ navigation, route: initialRoute }: Props) {
     parentAccount,
   }));
 
+  const isNFTSend = ["erc721.transfer", "erc1155.transfer"].includes(
```

Comment:
won't scale. shouldn't be on renderer side. => if not fixed now, we should mention this line of code as part of "what are the Ethereum specifics" LL ticket

Reply:
Helper is done and merged in https://github.com/LedgerHQ/ledger-live-common/pull/1543 
We'll have to add it once LLC has a new release. 👌